lib/oban/worker.ex

```diff
@@ -336,4 +338,12 @@ defmodule Oban.Worker do
   defp validate_opt!(option) do
     raise ArgumentError, "unknown option provided #{inspect(option)}"
   end
+
+  @spec to_string(module() | String.t()) :: String.t()
+  def to_string(worker) when is_atom(worker) and not is_nil(worker),
+    do: worker |> Kernel.to_string() |> to_string()
```

Comment:
Again, for consistency let's use a `do/end` block, there aren't any multi-line `do:` functions elsewhere in the code:

```suggestion
  def to_string(worker) when is_atom(worker) and not is_nil(worker) do
    worker
    |> Kernel.to_string()
    |> to_string()
  end
```